docs/modules/ROOT/pages/wan-replication.adoc

```diff
@@ -0,0 +1,79 @@
+= Configuring WAN Replication
+:description: pass:q[WAN Replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate a map from the source to the target Hazelcast cluster. ]
+
+{description}
+
+See xref:hazelcast:wan:wan.adoc[WAN replication] for more detailed information about the configuration and behavior of WAN replication in Hazelcast.
+
+== Prerequisites for WAN Replication
+
+Before you create and configure a `WanReplication` resource, you need to create the following:
+
+- A `Hazelcast` resource and `Map` resource for the source Hazelcast cluster.
+- A target Hazelcast cluster that is accessible from the source cluster.                
+== Configuring the WanReplication Resource
+
+
+You can use the following configuration options in your `WanReplication` resource. For more detailed information, see the xref:api-ref.adoc#wanreplicationspec[API Reference].
+
+[cols="20%m,80%a"]
```

Comment:
The field name on the second row of the table is wrapping so maybe we could adjust the column sizes to `[cols="25%m,75%a"]`?

Reply:
I will update it